lib/ledger_sync/error.rb

```diff
@@ -1,10 +1,43 @@
+# frozen_string_literal: true
+
 module LedgerSync
   class Error < StandardError
+    module HelpersMixin
+      module ClassMethods
+        def raise_if_unexpected_class(expected:, given:)
+          expected = [expected] unless expected.is_a?(Array)
```

Comment:
`expected = Array.wrap(expected)` I think basically does this, if you're interested in using that. Doesn't matter though